ci/launch_cluster.sh

```diff
@@ -62,11 +62,11 @@ function create-junit-xml {
 
 if ! ./dcos-launch create; then
   create-junit-xml "dcos-launch" "cluster.create" "Cluster launch failed."
-  exit 1
+  exit 0
```

Comment:
If a build returns 0 but JUnit finds and error it will report the build as unstable.

Reply:
Oh yes, this should be a comment in code. Otherwise the next guy will wonder.

Reply:
👍